src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+
+*** Test cases ***
+
+Test Columns-Titles and Resize
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    ${data-column-size_before2}=  Get Element Attribute  xpath=//*[@id="content"]/div[2]/div/div@data-column-size
```

Comment:
Remove the 2 from the name of this variable. Give the other variable a better name.